packages/python-packages/protocol-stub-generator/tests/test_serializer.py

```diff
@@ -0,0 +1,75 @@
+import os
+import sys
+import unittest
+import yaml
+from protocol.protocol_models import ProtocolClientView, ProtocolOperationView
+from protocol.parse_yml import create_python_name
+
+
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), ".."))
+
+
+PATH = os.getenv("TESTJSONPATH")
+
+
+class TestCases(unittest.TestCase):
+    def test_client_name(self):
+        with open(PATH) as f:
+            data = yaml.safe_load(f)
+        create_python_name(data)
+        client = ProtocolClientView.from_yaml(data)
+        assert client.Name == "Batch Document Translation Client"
+
+    def test_client_operationGroups(self):
+        with open(PATH) as f:
+            data = yaml.safe_load(f)
+        create_python_name(data)
+        client = ProtocolClientView.from_yaml(data)
+        groups = client.Operation_Groups
+        for g in groups:
+            assert g.operation_group == "DocumentTranslation"
+
+    def test_client_operation(self):
+        with open(PATH) as f:
+            data = yaml.safe_load(f)
+        create_python_name(data)
+        client = ProtocolClientView.from_yaml(data)
+        groups = client.Operation_Groups
+        for g in groups:
```

Comment:
is there only one group here? could we just do `assert len(groups[0].operations) == 9`